mlir/test/Examples/nvgpu/Ch2.py

```diff
@@ -0,0 +1,91 @@
+# RUN: env SUPPORT_LIB=%mlir_cuda_runtime \
+# RUN:   %PYTHON %s | FileCheck %s
+
+# ===----------------------------------------------------------------------===//
+#  Chapter 2 : 2D Saxpy with TMA
+# ===----------------------------------------------------------------------===//
+#
+# This program demonstrates 2D Saxpy. It is same as Chapter 1,
+# but it loads data using TMA (Tensor Memory Accelerator)
+#
+# This chapter introduces demonstrates:
+#  1. Create and initialize asynchronous transactional barrier (mbarrier)
+#  2. Execute Tensor Memory Accelerator (TMA) Load
+#  3. Wait for completion of TMA load with mbarrier
+#
+# ===----------------------------------------------------------------------===//
```

Comment:
I feel like these could be more complete descriptions so that it reads like literate program a bit more and we'd not need a separate set of docs.

Reply:
Improved the docs. This codes will go with my slides anyway, so we will have more docs